Sources/SyncDataProviders/Bookmarks/internal/SyncableBookmarkAdapter.swift

```diff
@@ -84,10 +92,15 @@ extension Syncable {
         if bookmark.isPendingDeletion {
             payload["deleted"] = ""
         } else {
-            if let title = bookmark.title {
-                payload["title"] = try encrypt(title)
-            }
             if bookmark.isFolder {
+                if let title = bookmark.title {
+                    if DDGSync.isFieldValidationEnabled {
+                        payload["title"] = try encrypt(String(title.prefix(BookmarkValidationConstraints.maxFolderTitleLength)))
+                    } else {
+                        payload["title"] = try encrypt(title)
+                    }
+                }
```

Comment:
I'm tempted to just release this as-is (ignoring the flag), as it doesn't really affect object graph.

Reply:
You specifically mean truncating folder titles, right? Just this?

Reply:
Yes, just truncating.

Reply:
Sure, it's done now.